MIGRATION.md

```diff
@@ -8,17 +8,67 @@ Pa11y's API changes between major versions. This is a guide to help you make the
 Table Of Contents
 -----------------
 
+- [Migrating from 4.0 to 5.0](#migrating-from-40-to-50)
 - [Migrating from 3.0 to 4.0](#migrating-from-30-to-40)
 - [Migrating from 2.0 to 3.0](#migrating-from-20-to-30)
 - [Migrating from 1.0 to 2.0](#migrating-from-10-to-20)
 
 
+Migrating from 4.0 to 5.0
+-------------------------
+
+**TODO this migration guide is a work in progress. Please update as you work on Pa11y 5.x**
+
+### PhantomJS to Headless Chrome
+
+Pa11y 5.0 switches from PhantomJS to [Headless Chrome](https://developers.google.com/web/updates/2017/04/headless-chrome). This allows us to use more modern JavaScript APIs and make Pa11y testing more stable.
+
+### Node.js Support
+
+Pa11y 5.0 now only supports Node.js v8.0.0 and higher, you'll need to upgrade to be able to use the latest versions of Pa11y.
+
+### Command-Line Interface
+
+The command-line interface in 5.0 is similar to 4.0, but there are a few key changes.
+
+  - The `--verify-page` flag has been removed, as page verification can be achieved with screen-shots or observing the debug output
+  - The `--htmlcs` flag has been removed, you can no longer configure the version of HTML CodeSniffer that Pa11y uses (TODO reconsider this)
```

Comment:
In two minds. On the one hand it'd be useful for people who want to upgrade/downgrade htmlcs (could this have saved us during our second-to-last release? 🤔 ). 

On the other hand maybe it'd be better to come up with an API of sorts so people can plug in $ENGINE and have it produce standard output.

Reply:
Yeah I think _maybe_ it's easier to remove this and readdress if somebody needs to switch out Phantom. Technically they still can but just with their own fork/build of Pa11y (if the need is dire and we don't have engine switching figured out.